src/views/VideoView/VideoView.style.tsx

```diff
@@ -82,3 +88,12 @@ export const LicenseContainer = styled.div`
     color: ${colors.gray[500]};
   }
 `
+
+export const TitleText = styled(Text)`
+  @media screen and (max-width: ${breakpoints.medium}) {
+    font-size: 32px;
```

Comment:
I think I'd be best to use values from `theme.typography.sizes` here and below. 32px will be `h3`